src/render_engine/cli/cli.py

```diff
@@ -410,6 +425,17 @@ def new_entry(
         # If `slug` is provided as a keyword add it to the `parsed_args` to be included in the rendering.
         # Prefer the keyword to what is passed via `--args`
         parsed_args["slug"] = slug
+    # Verify that we have a valid date should it be supplied or requested
+    if date := parsed_args.pop("date", None):
+        try:
+            date = dateparser.parse(date).isoformat()
+        except ParserError:
+            raise ValueError(f"Invalid date: {repr(date)}.") from None
+    elif include_date:
+        date = datetime.datetime.today().isoformat()
```

Comment:
How would tho work if the user is using the date but not the time. 

Is it possible to also add `date-only` which can also be set in the cli settings. (I'd also allow `include-date`)

Reply:
I see that you can just add the date and it zeroes out the time. 

Perhaps the `DATE_FORMAT` filter that was fixed will help with displaying date vs datetime. 

This should be documented heavily but that is a different issue.